pom.xml

```diff
@@ -55,7 +55,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-        <mainClass>com.homeaway.streamplatform.streamregistry.StreamRegistryApplication</mainClass>
+        <mainClass>StreamRegistryApplication</mainClass>
```

Comment:
Does this work without a fully qualified class name?

Reply:
Exactly! Like I was saying in the other comment!